terraform-cloud/aws/terraform.tfvars

```diff
@@ -0,0 +1,81 @@
+vpc_params = {
+  vpc1 = {
+    cidr_block = "10.0.0.0/16"
+    tags = {
+      Name : "vpc1"
+      owner : "owner1"
+    }
+  }
+  vpc2 = {
+    cidr_block = "11.0.0.0/16"
+    tags = {
+      Name : "vpc2"
+      owner : "owner2"
+    }
+  }
+  vpc3 = {
+    cidr_block = "12.0.0.0/16"
+    tags = {
+      Name : "vpc3"
+      owner : "owner3"
+    }
+  }
+}
+
+subnet_params = {
+  subnet11 = {
```

Comment:
Smal inconsistency between names of the subnets and names of the resources. We should make it consistent.

Reply:
Done.